From 1ad2b3aae3ec1edf9310932d104073d0b9ae35cd Mon Sep 17 00:00:00 2001
From: Luca Weiss <luca@lucaweiss.eu>
Date: Sat, 28 Jun 2025 16:52:19 +0200
Subject: [PATCH] Bump minimum required CMake version

CMake 4.0.3 does not build this project anymore due to "Compatibility
with CMake < 3.5 has been removed from CMake.".

Bump the minimum version to 3.16, being the CMake version from Ubuntu
20.04.
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 08069f1..51190de 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.4.0)
+cmake_minimum_required(VERSION 3.16.0)
 project(lomiri-schemas LANGUAGES NONE)
 
 set(PACKAGE ${CMAKE_PROJECT_NAME})
-- 
2.50.0

